mne/epochs.py

```diff
@@ -1502,8 +1514,13 @@ def __repr__(self):
         if self.baseline is None:
             s += 'off'
         else:
-            s += '[%s, %s]' % tuple(['None' if b is None else ('%g' % b)
-                                     for b in self.baseline])
+            s += '[%s, %s] sec' % tuple(['None' if b is None else ('%g' % b)
+                                        for b in self.baseline])
+            if self.baseline != _check_baseline(
+                    self.baseline, times=self.times, sfreq=self.info['sfreq'],
+                    on_baseline_outside_data='adjust'):
+                s += ' (baseline period was cropped after baseline correction)'
```

Comment:
I need to rethink this one, as we still have a float comparison here

Reply:
No never mind, I think this is okay in this specific case… emphasis on I **think**

Reply:
`_check_baseline` allows for up to `1./sfreq` slack, so this should be okay here.